src/styles/components/_calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  width: ($day-size * $week-long) + ($cell-gap * ($week-long - 1));
+  padding: 10px;
+  gap: $cell-gap;
+  flex-wrap: wrap;
+  list-style: none;
+
+  @each $day, $day-num in $week-days {
+    &--day-start--#{$day}>:first-child {
+      margin-left: ($day-size + $cell-gap) * $day-num;
+    }
+  }
+
+  @for $day from $min-month-long through $num-days {
+    &--month-length--#{$day}>:NOT(:nth-child(-n+#{$day})) {
```

Comment:
`>`  is not necessaary here

Reply:
Removed ">" as well, for some reason it didn't work without this sign when I first developed this section...

Reply:
doesn't work without it as well